lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+			if (obj.name.match(patternDeprecated) === null) {
+				let tokenName = obj.name;
+				tokenName = tokenName.replace(/\//i, '-');
+
+				obj.values.hex = obj.values.hex.replace(/FFFFFF/gi, function(){
+					return 'fff';
+				});
+
+				if (obj.values.alpha !== 1) {
+					result.push('$' + tokenName + ': ' + 'rgba(var(' + hexToCssVariable[obj.values.hex] +'), ' + obj.values.alpha + ');');
+				} else {
+					result.push('$' + tokenName + ': ' + hexToCssVariable[obj.values.hex] + ';');
+				}
+
+
+
+				/** JSON object structure */
+				const tokenNameSplitted = tokenName.split('-');
+				const category = tokenNameSplitted[1];
+				const resultJsonObject: ITokenJsonObject = {
+					id: binValues.destination + '_' + tokenName,
+					label: tokenName,
+					value: hexToCssVariable[obj.values.hex],
+				};
+				if (resultJson[category] === undefined) {
+					resultJson[category] = [];
+				}
+				resultJson[category].push(resultJsonObject);
+			}
+		});
+		saveTokenFile(tokensFilesConfig.destinationPath + binValues.destination, result);
+		saveTokenFileJSON(tokensFilesConfig.destinationPath + binValues.destinationJson, resultJson);
+	} catch (err) {
+		console.error(err)
+	}
+}
+
+const saveTokenFileJSON  = (filepath: string, content: any) => {
+	let file = fsT.createWriteStream(filepath);
+	file.on('error', function(err) { console.log(err) });
+	file.write(JSON.stringify(content));
+	file.end();
+}
+
+const saveTokenFile = (filepath, content) => {
+	let file = fsT.createWriteStream(filepath);
+	file.on('error', function(err) { console.log(err) });
+	content.forEach(function(v) { file.write(v.toLowerCase() + '\n'); });
+	file.end();
+}
+
+const hexToRgb = (hex: string ) => {
+	if (hex.length === 4) {
+		hex = hex + hex[1] + hex[2] + hex[3];
+	}
+	let result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+	return result ? '' + parseInt(result[1], 16) + ', ' +
+		parseInt(result[2], 16) + ', ' + parseInt(result[3], 16)  : null;
+}
+
+const saveColorFile = (filepath: string, content: any) => {
+	let file = fsT.createWriteStream(filepath);
+	file.on('error', function(err) { console.log(err) });
+	file.write(':root {\n');
+	content.forEach(function(v) { file.write('\t' + v.toLowerCase() + '\n'); });
+	file.write('}\n')
+	file.end();
+}
+
+const saveColorFileJSON  = (filepath: string, content: any) => {
+	let file = fsT.createWriteStream(filepath);
+	file.on('error', function(err) { console.log(err) });
+	file.write(JSON.stringify(content));
+	file.end();
+}
+
+const AllInOneImporter = () => {
+	tokensFilesConfig.bins.forEach(bin =>{
+		axios.get(tokensFilesConfig.jsonBinApiUrl + bin.id + '/latest')
+		.then(function(response) {
+			let hexToCssVariable;
+			Object.keys(bin.files).forEach(key => {
+				switch (key) {
+					case "colorsRaw":
+						hexToCssVariable = ImportColorsRaw(bin.files[key], response);
+						break;
+					case "tokens":
+						ImportSingleTokenFile(bin.files[key], response, hexToCssVariable);
```

Comment:
this one is tricky. The order is dependent on the config file, on the other hand, if you call `ImportSingleTokenFile()` before `ImportColorsRaw()` you'll get an empty variable. IMO, we should mitigate this problem somehow

Reply:
Can You check now is it ok?